x/migration/types/message_claim_morse_pokt_test.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+import (
+	"encoding/hex"
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+
+	"github.com/pokt-network/poktroll/testutil/sample"
+)
+
+func TestMsgClaimMorsePokt_ValidateBasic(t *testing.T) {
+	tests := []struct {
+		name string
+		msg  MsgClaimMorsePokt
+		err  error
+	}{
+		{
+			name: "invalid source address",
+			msg: MsgClaimMorsePokt{
+				MorseSrcAddress:    "invalid_address",
+				ShannonDestAddress: sample.AccAddress(),
+			},
+			err: sdkerrors.ErrInvalidAddress,
+		}, {
+			name: "invalid address",
+			msg: MsgClaimMorsePokt{
+				MorseSrcAddress:    hex.EncodeToString(sample.ConsAddress().Bytes()),
```

Comment:
Do you mind moving these into local vars and adding some comments to the fact that

Consensus Address == Ed25519 == Morse

Feel free to link to https://www.notion.so/buildwithgrove/Background-Addresses-Key-Schemes-175a36edfff680ff8f23fc7599fd9107?pvs=4